LICENSE

```diff
@@ -1,5 +1,7 @@
 MIT License
 
+Copyright (c) 2017 Nathan Sweet
+Copyright (c) 2018, 2019 Cloudflare
 Copyright (c) 2019 Cilium
```

Comment:
Minor nit, For Cilium we usually use "Copyright yyyy Authors of Cilium". Not strictly related to this PR but adjacent so may as well fix it.